src/dotnet/APIView/APIViewWeb/Languages/XmlLanguageService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+
+namespace APIViewWeb
+{
+    public class XmlLanguageService : LanguageProcessor
```

Comment:
Can we inherit Java Language processor instead of LanguageProcesor? Otherwise it will become an issue if someone forget to update the jar version in one place.

Reply:
I wanted to do this too, but I don't have my system setup to do c# development. If someone wants to improve this PR to do it properly, I would appreciate it

Reply:
I can try to push some changes if that is okay with you.

Reply:
yes, go for it!